File: c/1013_the_greatest.c
/*  
    @autor: Guiventura6;
    @data: 20/01/2022;
    @Problema: The Greatest;        
*/
#include <stdio.h>

int main(){

    int a, b, c;

    scanf("%d%d%d", &a, &b, &c);

    int maiorAB = (a + b + abs(a -b)) / 2;
    
    int theGreatest = maiorAB;
    if (theGreatest < c)
    {
        theGreatest = c;
    }

    printf("%d eh o maior\n", theGreatest);

    return 0;
}

File: python/1013_the_greatest.py
'''
    Autor: Guiventura6;
    Data: 02/03/2022
    Modified: 27/06/2023
    Problema: The Greatest
    Resume: Program that reads three integer values and present the greatest
'''
a, b, c = map(int, input().split())
maiorAB = (a + b + abs(a - b)) // 2

if maiorAB > c:
    print('{} eh o maior'.format(maiorAB))
else:
    print('{} eh o maior'.format(c))
